src/defines.h

```diff
@@ -23,8 +23,8 @@
 #define BALL_RADIUS (25.0f * COORDINATE_SCALING)
 #define BALL_START_POS_X (PADDLE_START_POS_X + PADDLE_WIDTH / 2.0f)
 #define BALL_START_POS_Y (PADDLE_START_POS_Y + BALL_RADIUS)
-#define BALL_TRANSLATION_X (1.0f * COORDINATE_SCALING)
-#define BALL_TRANSLATION_Y (1.0f * COORDINATE_SCALING)
+#define BALL_START_TRANSLATION_X (0.0f * COORDINATE_SCALING)
+#define BALL_START_TRANSLATION_Y (200.0f * COORDINATE_SCALING)
```

Comment:
I think it should be BALL_LAUNCH_TRANSLATION_X and BALL_LAUNCH_TRANSLATION_X now